docs/Authenticate and call Fraud Protection.md

```diff
@@ -45,7 +45,7 @@ public class TokenProviderService : ITokenProvider
     {
         var clientAssertion = new ClientCredential(_settings.ClientId, _settings.ClientSecret);
         var context = new AuthenticationContext(_settings.Authority);
-        var authenticationResult = await context.AcquireTokenAsync(resource, clientAssertion);
+        var authenticationResult = await context.AcquireTokenAsync(_settings.resource, clientAssertion);
```

Comment:
Isn't it _settings.Resource, not _settings.resource?
Applies for line 39 as well.

Reply:
Great catch.  Thanks.